revista-brasileira-de-ciencia-do-solo.csl

```diff
@@ -47,432 +47,217 @@
-            <date-part name="day"/>
-            <date-part prefix=" " name="month" form="short"/>
-            <date-part prefix=" " name="year"/>
-          </date>
-          <text variable="section" prefix="Sec. "/>
-          <text variable="page" prefix="p. " suffix="."/>
-        </group>
+      <if type="article-journal article-magazine" match="any">
+        <text variable="volume"/>
       </if>
-      <else-if match="any" type="article-journal article-magazine article-newspaper">
-        <group>
-          <group>
-            <text variable="volume" suffix=":"/>
-          </group>
-          <text variable="page"/>
-        </group>
-      </else-if>
-      <else-if match="any" type="book chapter">
-        <group delimiter=", ">
-          <group>
-            <text variable="volume" prefix="v. "/>
-            <text variable="page" prefix="p. "/>
-          </group>
-        </group>
-      </else-if>
     </choose>
   </macro>
-  <macro name="collection-title">
-    <text variable="collection-title"/>
-    <text variable="collection-number" prefix=" "/>
-  </macro>
-  <macro name="genre">
-    <text variable="genre"/>
-  </macro>
-  <macro name="citation-locator">
-    <group>
-      <label variable="locator" form="short"/>
-      <text variable="locator" prefix=" "/>
-    </group>
-  </macro>
-  <macro name="place">
+  <macro name="report-details">
     <choose>
-      <if match="any" variable="publisher-place">
-        <text variable="publisher-place"/>
+      <if type="report">
+        <text variable="number" prefix="Report No.: "/>
       </if>
     </choose>
   </macro>
-  <macro name="archive">
-    <group>
-      <text variable="archive" prefix=" "/>
-    </group>
+  <macro name="author-short">
+    <names variable="author" suffix=",">
+      <name form="short" delimiter=" e " sort-separator=","/>
+    </names>
   </macro>
-  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-names="false" disambiguate-add-givenname="false" collapse="year-suffix">
+  <macro name="issued">
+    <date variable="issued">
+      <date-part name="year"/>
+    </date>
+  </macro>
+  <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" collapse="year">
     <sort>
-      <key macro="author"/>
-      <key variable="issued"/>
+      <key macro="issued"/>
+      <key macro="author-short"/>
     </sort>
     <layout prefix="(" suffix=")" delimiter="; ">
-      <group>
-        <text macro="author-short" text-case="capitalize-first" suffix=", "/>
-        <text macro="issued-year"/>
-        <text prefix=", " macro="citation-locator"/>
+      <group delimiter=" ">
+        <text macro="author-short"/>
+        <text macro="issued"/>
       </group>
     </layout>
   </citation>
-  <bibliography hanging-indent="false" et-al-min="3" et-al-use-first="1" entry-spacing="1">
+  <bibliography et-al-min="0" et-al-use-first="0" entry-spacing="0" line-spacing="0">
```

Comment:
`et-al-min="0" et-al-use-first="0"`

This doesn't make any sense. This way no author names are printed.